corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -140,6 +140,11 @@ def build_command(self):
             ),
         )
 
+    def build_title(self):
+        return Display(
```

Comment:
This uses the `Display` class, but the XML in tests doesn't have a `<display>` block, do you know why that is?